server/controllers/songsController.js

```diff
@@ -1,39 +1,49 @@
 import asyncHandler from "../middleware/asyncHandler.js";
 import Song from "../models/Song.js";
+import logger from "../logger.js";
 import ErrorResponse from "../utils/ErrorResponse.js";
 import createObjectFromQuery from "../utils/createObjectFromQuery.js";
 import { getOrCreateArtist } from "./artistsController.js";
 import { createDummySong } from "../utils/createDummyData.js";
 
 const findSong = async (req) => {
   const filter = createObjectFromQuery(req.body);
+  logger.info(`findSong with song details ${filter} sent`);
   const songsArray = await Song.find(filter).populate("artist");
   if (songsArray.length > 0) return songsArray;
 };
 
 const createSongInDB = async (req) => {
   const newSongObject = createObjectFromQuery(req.body);
+  logger.info(`createSongInDB with song details ${newSongObject}`);
 
   //Add a function here that scrapes the song, translates it and returns the information bellow (name, lyrics, album...) in one object.
   const dummySong = createDummySong(newSongObject);
   const data = dummySong;
+  logger.info(`new song created  with song details ${data}`);
```

Comment:
does this log the data or '[Object object]'?

Reply:
data